lib/urlwatch/main.py

```diff
@@ -59,8 +59,11 @@ def __init__(self, urlwatch_config, config_storage, cache_storage, urls_storage)
         if hasattr(self.urlwatch_config, 'migrate_urls'):
             self.urlwatch_config.migrate_urls(self)
 
-        self.load_hooks()
-        self.load_jobs()
+        if not getattr(self.urlwatch_config, 'edit_hooks', False):
```

Comment:
Thanks! That was my original solution but it wasn't passing the tests, maybe it was not an issue?

Reply:
It shouldn't be an issue for the tests. Did you check the error messages of the tests?

Reply:
It would say
```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.4.6/lib/python3.4/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/thp/urlwatch/test/test_handler.py", line 124, in test_run_watcher
    urlwatcher = Urlwatch(urlwatch_config, config_storage, cache_storage, urls_storage)
  File "/home/travis/build/thp/urlwatch/lib/urlwatch/main.py", line 62, in __init__
    if not self.urlwatch_config.edit_hooks:
nose.proxy.AttributeError: 'TestConfig' object has no attribute 'edit_hooks'
```

just a bad/different test config?

Reply:
Hmmm... The tests use `TestConfig` instead of the regular `config.CommandConfig`. The latter does command line parsing and add all attributes and defaults, while the former one doesn't. Maybe you can modify the tests to make it work better.